autowebcompat/network.py

```diff
@@ -185,23 +187,29 @@ def create_inception_network(input_shape):
     return Model(input, x)
 
 
-def create_resnet50_network(input_shape):
-    base_model = ResNet50(input_shape=input_shape)
+def create_resnet50_network(input_shape, weights):
+    base_model = ResNet50(input_shape=input_shape, weights=weights)
     return Model(inputs=base_model.input, outputs=base_model.get_layer('flatten_1').output)
 
 
 def create(input_shape, network='vgglike', weights=None):
     assert network in SUPPORTED_NETWORKS, '%s is an invalid network' % network
+    if weights is not None:
+        assert network in SUPPORTED_NETWORKS_WITH_WEIGHTS, '%s does not have weights for %s ' % (network, weights)
```

Comment:
When `weights` is a file, we shouldn't check if the network is in SUPPORTED_NETWORKS_WITH_WEIGHTS, as it could be any of the networks